sycl/test-e2e/SharedLib/use_when_link_new_offload.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: linux
+// REQUIRES: opencl, cpu, linux
```

Comment:
This test is expected to check if the shared library support works with new offload model. An extensive testing for the new offloading model for all targets will be accomplished later.

Thanks